content/release-10-18-0/datahub-10-18-0-bundle/10_18_0.md

```diff
@@ -0,0 +1,45 @@
+
+### Efficient ODBC connectivity
+
+The ODBC driver required for running queries against Dremio now relies on Apache Arrow, whose columnar memory format allows for a more efficient ODBC-based query execution.
+
+### Enforcement of single data types
+
+Columns having multiple data types, so-called mixed types, can no more occur. During the offloading process, Cumulocity IoT DataHub takes care that the offloaded data has one type per column. If incoming data might lead to multiple types, either the associated offloading pipeline will be stopped or the resulting schema is automatically evolved.
+
+### Improved handling of additional columns
+
+The UI simplifies duplicating an additional column and changing the data type of the column.
+
+### Limitations
+
+|<div style="width:250px">Description</div>
+|:---
+|Mixed usage of uppercase and lowercase characters for attribute names in the documents is not supported.|
+|If the collection to be offloaded has JSON attributes consisting of more than 32,000 characters, its data cannot be offloaded. One specific case where this limitation applies is Cumulocity IoT's application builder, which stores its assets in the inventory collection when being used.|
+|If the collection to be offloaded has more than 800 JSON attributes, its data cannot be offloaded. This limitation also includes nested JSON content, which will be expanded into columns during offloading. Therefore, measurements documents with more than 800 series/series value fragments are not supported.|
+|In previous releases, the *childDevices* and *childAssets* fields were part of the default offloading columns for the inventory collection. They are now excluded from the offloading process as their potentially high number of list items leads to problems. When that number exceeds a corresponding Dremio limit, the offloaded data is no longer readable. New inventory offloading configurations exclude those two fields. For existing inventory configurations, they are automatically added as additional result columns, in order to preserve the set of columns handled by the offloading configuration.|
+|When using Microsoft Azure as data lake with IP white-listing being activated, Cumulocity IoT DataHub cannot access the data lake.|
+
+### Known issues
+
+|<div style="width:250px">Edition|Description|
+|:---|:---|
+|Cloud|Data lake configuration validation is broken in terms of wrong bucket names (AWS S3) and wrong account names (Azure Storage). When saving the settings with an invalid bucket/account name, DataHub fails to quickly detect the problem and will instead run a time-consuming check, which shows up as an ongoing save request in the UI. Eventually the request will fail in the UI with a timeout and the save request in the backend will fail as well. In such a case, please carefully check the bucket/account name and try saving again.|
```

Comment:
```suggestion
|Cloud|Data lake configuration validation is broken in terms of wrong bucket names (AWS S3) and wrong account names (Azure Storage). When saving the settings with an invalid bucket/account name, DataHub fails to quickly detect the problem and will instead run a time-consuming check, which shows up as an ongoing save request in the UI. Eventually, the request will fail in the UI with a timeout and the save request in the back end will fail as well. In such a case, please carefully check the bucket/account name and try saving again.|
```

Reply:
Comma is fixed; backend is again a standard term in this context.

So all in all we should be done...